README.md

```diff
@@ -49,7 +49,7 @@ bili
 Then you will get `./dist/index.cjs.js`. To generate in more formats, try:
 
 ```bash
-bili --format cjs,es,umd,umd-min
```

Comment:
ah this is actually valid too, just a shortcut for `--format cjs --format es --format umd --format umd-min`